src/client/java/teammates/client/scripts/util/LoopHelper.java

```diff
@@ -0,0 +1,39 @@
+package teammates.client.scripts.util;
+
+/**
+ * Utility class to be used with loops to print messages regularly.
```

Comment:
Apparently, people use `utility class` to refer to classes with only static methods. May be `Helper class`?